src/main/java/qna/domain/BaseEntity.java

```diff
@@ -21,8 +21,14 @@ public abstract class BaseEntity {
     @LastModifiedDate
     private LocalDateTime updatedAt;
 
+    public BaseEntity(){
```

Comment:
접근제어자를 고민해 보면 어떨까요? 😄 
```suggestion
    protected BaseEntity() {
```

Reply:
넵!! 생성자 접근제어자도 적절히 맞춰 쓰도록 하겠습니다!! 😄